src/Modules/Workspaces/Commands.Workspaces/help/Set-PowerBIWorkspace.md

```diff
@@ -1,7 +1,7 @@
 ---
 external help file: Microsoft.PowerBI.Commands.Workspaces.dll-Help.xml
 Module Name: MicrosoftPowerBIMgmt.Workspaces
-online version:
+online version: https://docs.microsoft.com/en-us/powershell/module/microsoftpowerbimgmt.workspaces/set-powerbiworkspace?view=powerbi-ps-1.0.193
```

Comment:
You consider dropping the version number from the URL so we don't have to update this with each release.

Reply:
It's a temporary thing, we are getting a version moniker of 'latest' so we can supply that but we have to supply something currently (it errors if we leave off the view). It's probably a few weeks out to get the new version moniker in MicrosoftDocs